packages/babel-preset-pwa-kit/.eslintrc.js

```diff
@@ -1,15 +1,10 @@
 /*
```

Comment:
I don't know why git thinks that this was a moved/renamed file. `$.eslintrc.js` was deleted and this is a new file. They're not related.